client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -341,17 +346,25 @@ class TransferDomainPrecheck extends React.Component {
 							) }
 						</p>
 					</div>
-					<Button
-						disabled={ false === unlocked || ! authCodeValid || currentStep < 3 }
-						onClick={ this.onClick }
-						primary={ true }
-					>
+					<Button disabled={ disableButton } onClick={ this.onClick } primary={ true }>
 						{ translate( 'Continue' ) }
 					</Button>
 				</Card>
 			</div>
 		);
 	}
+
+	supportUserNotice() {
+		if ( isSupportUserSession() ) {
+			return (
+				<Notice
+					text={ this.props.translate( 'Only users can submit the actual transfer.' ) }
```

Comment:
Hmm, this copy might be a bit confusing if the HE does not know the context. Maybe let's be a bit more explicit, like `Transfers cannot be initiated in a support session - please ask the user to do it instead.`

Reply:
You win. :stuck_out_tongue: